config/prod/cmc-migration-bucket.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.8.4/templates/S3/synapse-external-bucket.j2"
+stack_name: "cmc-migration-bucket"
+stack_tags:
+  OwnerEmail: 'thomas.yu@sagebase.org'
+  CostCenter: "UMass PECDCC / 106000"
+parameters:
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  SameRegionResourceAccessToBucket: 'true'
```

Comment:
I'm assuming we will want to restrict downloads to us-east-1, but will confirm on slack